word-level-pos-tokens-recogs-style-decoder-loop.rasp

```diff
@@ -934,14 +990,15 @@ no_pp_np_mask = 1 - aggregate((pp_one_after_mask and np_prop_diag_mask) or (pp_t
 nps_without_pp_prefix_indices = selector_width(select(NOUN_MASK*no_pp_np_mask, 1, ==) and select(indices, indices, <=))*NOUN_MASK*no_pp_np_mask;
 
 left_idx = aggregate(select(indices, left_idx_in_nps_zero_based, ==), input_indices_sorted);
-right_idx = aggregate(select(nps_without_pp_prefix_indices, after_intro_idx, ==), indices);
+right_idx = aggregate(select(nps_without_pp_prefix_indices, after_intro_idx+cp_in_output_count, ==), indices);
```

Comment:
Here is an example of where "pp np" is already skipped when getting a relationship index